src/utils.cpp

```diff
@@ -1102,6 +1102,11 @@ int utils::run_interactively(
 	} else if (status == 127) {
 		LOG(level::DEBUG, "%s: couldn't run shell", caller);
 	}
+	if (status != 0) {
+		LOG(level::DEBUG, "utils::run_interactively: "
```

Comment:
Other log messages in this function don't include "utils::run_interactively" in the message, so this code shouldn't include it, too.

I don't have a strong rationale for this behaviour, though.